plugins/base/src/main/kotlin/transformers/pages/tags/SinceKotlinTagContentProvider.kt

```diff
@@ -0,0 +1,38 @@
+package org.jetbrains.dokka.base.transformers.pages.tags
+
+import org.jetbrains.dokka.DokkaConfiguration
+import org.jetbrains.dokka.base.translators.documentables.PageContentBuilder.DocumentableContentBuilder
+import org.jetbrains.dokka.model.doc.CustomTagWrapper
+import org.jetbrains.dokka.pages.ContentKind
+import org.jetbrains.dokka.pages.TextStyle
+
+object SinceKotlinTagContentProvider : CustomTagContentProvider {
+
+    private val TAG_NAME = "Since Kotlin"
+
+    override fun DocumentableContentBuilder.contentForDescription(
+        sourceSet: DokkaConfiguration.DokkaSourceSet,
+        customTag: CustomTagWrapper
+    ) {
+        sinceKotlinBlock(sourceSet, customTag)
+    }
+
+    override fun DocumentableContentBuilder.contentForBrief(
+        sourceSet: DokkaConfiguration.DokkaSourceSet,
+        customTag: CustomTagWrapper
+    ) {
+        sinceKotlinBlock(sourceSet, customTag)
```

Comment:
Yeah, I agree! Actually, not only here, but in the description block as well

If you don't mind, I'd like to keep this PR as extension point only (so leave the old behaviour/UI to make sure nothing's broken), and we can change "Since Kotlin" format as a separate issue (#2359) with additional discussion right after.

Reply:
> but in the description block
It should be consistent with another description blocks.
 
I suppose everyone can use [SinceKotlin](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-since-kotlin/) annotation.